SYCL/ESIMD/regression/Inputs/complex-lib-esimd.cpp

```diff
@@ -0,0 +1,26 @@
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/experimental/esimd.hpp>
+
+static constexpr const int VL = 4;
+
+sycl::event add(size_t n, sycl::buffer<int, 1> &buf_a,
+                sycl::buffer<int, 1> &buf_b, sycl::buffer<int, 1> &buf_c,
+                sycl::queue &Q) {
+  auto E = Q.submit([&](sycl::handler &H) {
+    sycl::accessor acc_a{buf_a, H, sycl::read_only};
+    sycl::accessor acc_b{buf_b, H, sycl::read_only};
+    sycl::accessor acc_c{buf_c, H, sycl::write_only};
+
+    H.parallel_for(n, [=](sycl::id<1> i) SYCL_ESIMD_KERNEL {
+      using namespace sycl::ext::intel::experimental::esimd;
+      unsigned int offset = i * VL * sizeof(int);
+      simd<int, VL> va;
+      va.copy_from(acc_a, offset);
```

Comment:
Nit: here and in few other places - there is a convenience constructor:
```suggestion
      simd<int, VL> va(acc_a, offset);
```

Reply:
Applied in c18962fa70c8900fd6aa9022cd3f75327ebe137d